sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -123,6 +125,27 @@ public void createItemEncrypt_readItemDecrypt() {
         validateResponse(properties, responseItem);
     }
 
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void createItemEncryptWithContentResponseOnWriteEnabledFalse() {
+        CosmosAsyncClient asyncClient = null ;
+        try {
+            asyncClient =
```

Comment:
you don't need to create a new client to change `ContentResponseOnWriteEnabled`.

You can set `ContentResponseOnWriteEnabled` per operation. the benefit is you don't create a new client in the test .
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java#L261-L264


Generally we should try to re-use the clients when possible as client instantiation is time costly on the test. (will add more time to the test time). Not a big deal anyway.

Reply:
done